src/services/api.ts

```diff
@@ -68,7 +68,7 @@ type ReadRepresentativePointsResponse = {
     2.5?: number
     5: number
   }
-  service_area_id: number
```

Comment:
Can't we pull that from datatypes?

Reply:
Intentionally keeping API types and local model types separate, so that we need to update one then the other explicitly. Otherwise, we might get the illusion that API and local model types are in sync when they are not.